File: src/main/java/failchat/funstream/SourceFilter.java
package failchat.funstream;

import failchat.core.MessageFilter;

/**
 * Фильтруент сообщения с других источников
 * */
public class SourceFilter implements MessageFilter<FsMessage> {
    @Override
    public boolean filterMessage(FsMessage message) {
        return message.getFrom().getId() < 0 || message.getTo() == null;
    }
}

File: src/main/kotlin/failchat/peka2tv/SourceFilter.kt
package failchat.peka2tv

import failchat.core.chat.MessageFilter

/**
 * Фильтруент сообщения с других источников
 */
class SourceFilter : MessageFilter<Peka2tvMessage> { //todo rename?
    override fun filterMessage(message: Peka2tvMessage): Boolean {
        return message.fromUser.id < 0
    }
}
